src/txmempool.h

```diff
@@ -672,10 +672,10 @@ class CTxMemPool
 class CCoinsViewMemPool : public CCoinsViewBacked
 {
 protected:
-    CTxMemPool &mempool;
+    const CTxMemPool &mempool;
 
 public:
-    CCoinsViewMemPool(CCoinsView *baseIn, CTxMemPool &mempoolIn);
+    CCoinsViewMemPool(CCoinsView *baseIn, const CTxMemPool &mempoolIn);
```

Comment:
nit: If you do trivial fixes, please also do all other trivial fixes in the same line:

```diff
diff --git a/src/main.cpp b/src/main.cpp
index ed34137..2777245 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -1471 +1471 @@ bool GetTransaction(const uint256 &hash, CTransaction &txOut, const Consensus::P
-            const CCoinsViewCache &view = *pcoinsTip;
+            const CCoinsViewCache& view = *pcoinsTip;
diff --git a/src/txmempool.h b/src/txmempool.h
index c991514..d17e332 100644
--- a/src/txmempool.h
+++ b/src/txmempool.h
@@ -675 +675 @@ protected:
-    const CTxMemPool &mempool;
+    const CTxMemPool& mempool;
@@ -678 +678 @@ public:
-    CCoinsViewMemPool(CCoinsView *baseIn, const CTxMemPool &mempoolIn);
+    CCoinsViewMemPool(CCoinsView* baseIn, const CTxMemPool& mempoolIn);